themes/default.theme/input.sass

```diff
@@ -4,3 +4,42 @@ input
     border
       color: $divider-color
       width: 1px
+  keyInput
+    background: white
+    inactive
+      margins: 40
+      color: "black"
+      background: $divider-color
+      animation
+        marginAnimation
+          duration: 300
+        colorAnimation
+          duration: 600
+    active
+      margins: 20
+      color: "black"
```

Comment:
I think white would look better ;)
And definitely use a larger font size otherwise the text looks so lost